src/pattern-matching/infallible.md

```diff
@@ -37,8 +37,8 @@ fn main() {
 - Patterns are type-specific, including irrefutable patterns. Try adding or
   removing an element to the tuple and look at the resulting compiler errors.
 
-- Variable names are patterns that always match and which bind the matched value
-  into a new variable with that name.
+- Variable names are patterns that always match and bind the matched value into
+  a new variable with that name.
```

Comment:
I think the previous text was clearer.

Reply:
Both are equally fine imo. The presence of "which" doesn't change the meaning of the sentence, and I think it's down to personal taste whether you like the "which" or not.